docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Live Simulation Mode
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Live Simulation Mode
+
+Forget screen sharing, there’s live simulation mode! Let’s say you’re working on the application logic and want a second opinion or to share your machine with your team mates, live simulation has made this process as easy as sharing a link. It is not necessary to take screenshots anymore because this feature will let all users with the specially generated link to access and discuss in real time the process of your machine.
+
+In order to use this collaborative feature, the user initiating the live simulation must be signed in. When looking at the Stately Studio, you’re able to switch between two views: Design mode and Simulate mode.
+
+:::studio
+
+Live simulation is a Pro feature of the Studio. [Check out the features on our Pro plan](/studio-pro-plan.mdx) or [upgrade to try the Pro plan with a 30-day free trial](https://stately.ai/registry/billing).
+
+:::
+
+## Simulate your machine with others
+
+1. Select the **Simulate** mode from the center of the visual editor. When in the **Simulate** mode, you’ll see your machine with the initial state turn the color blue. Blue means your machine is not currently being edited or shared, but you are simulating the logic flow of the current machine. In **Simulate** mode, this is where you can go through each state and transition to thoroughly review your logic.
```

Comment:
We can make the steps easier to skim-read by making them as short as possible, and pulling out any supporting information into further tips and sentences. I think this supporting info would make a good tip.

We also need to bear in mind that not everyone can perceive colour, so we can’t rely on it as the only way to let a person know they’re in Simulate mode. So I might also mention that it’s highlighted:

```suggestion
1. Select the **Simulate** mode from the center of the visual editor. 

:::tip

When you’re in **Simulate** mode, your current state and next available events are highlighted in blue to simulate the flow of your machine. Click the available events to transition through states and review your logic.

:::
```